source/server/hot_restart_impl.cc

```diff
@@ -43,7 +43,7 @@ SharedMemory& SharedMemory::initialize(uint32_t stats_set_size, Options& options
   const uint64_t total_size = sizeof(SharedMemory) + stats_set_size;
 
   int flags = O_RDWR;
-  const std::string shmem_name = fmt::format("/envoy_shared_memory_{}", options.baseId());
+  const std::string shmem_name = fmt::format("/envoy_block_memory_{}", options.baseId());
```

Comment:
I think this one should be reverted. a) it is shared memory. b) it's possible people have setup permissions to allow this exact path. I see no reason to change it...

Reply:
Good catch, I'll swap this back.